content/general/concepts/semantic-technologies/terms/rdf.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: 'rdf'
+Description: 'A language for representing semantic data.'
+Subjects:
+  - 'Data Science'
+Tags:
+  - 'Data'
+  - 'Semantic
+  - 'Metadata'
+  - 'Tuples'
+CatalogContent:
+  - 'paths/data-science-foundations'
+  - 'paths/data-science-nlp'
+---
+
+**Resource Description Framework (RDF)** is a language for representing information. All data within the suite of semantic technologies is represented in RDF.
+
+## RDF Triples
+
+RDF represents data in the format of **triples** -- ordered sets of three elements. The elements in an RDF triple stand for a subject, a predicate, and an object. An RDF triple can therefore be represented as follows:
+
+```pseudo
+subject, predicate, object
+```
+
+- *`subject`*: The element which the triple says something about.
+- *`predicate`*: A feature or property that applies to `subject`.
+- *`object`*: The value of the feature or property represented by `predicate`.
+
+To illustrate, take the sentence 'The novel *Pachinko* was written by Min Jin Lee'. The sentence conveys something about *Pachinko*: that it has a feature of having an author, and that this author is Min Jin Lee. This information can be represented by the following triple:
+
+```pseudo
+Pachinko, hasAuthor, Min Jin Lee
+```
+
+### Triple Terms
+
+In contrast to the above examples, the elements in a valid RDF triple cannot just be any values at all, such as arbitrary strings (e.g., `hasAuthor`). Rather, each of these elements can only be particular values:
+
+- A `subject` is either an **International Resource Identifier (IRI)**, a **blank node**, or a **quoted triple**.
+- A `predicate` must be an IRI.
+- An `object` is either an IRI, a blank node, a quoted triple, or a **literal**.
+
+Each of these terms is explained below:
```

Comment:
Please add an example like you initially added while introducing triple.

Reply:
@vaibhavmani810 Are you asking for examples of these terms ('IRI', 'blank node', etc.), or an example of a triple using these kinds of terms?

Reply:
Example of a Triple using these kind of terms.